docs/general/networking/vpn.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: network-vpn
+title: VPNs
+---
+
+# VPNs
+
+"A virtual private network (VPN) is a mechanism for creating a secure connection between a computing device and a computer network, or between two networks, using an insecure communication medium such as the public Internet." - [Wikipedia](https://en.wikipedia.org/wiki/Virtual_private_network)
+
+## Commercial VPN services
+
+Commercial VPN services such as PIA, NordVPN or Windscribe are a great way to hide your IP and bypass georestrictions. While some of them do provide a port forwarding feature, hosting Jellyfin using one is not recommended.
+
+## Point To Point VPNs
+
+Services like Tailscale and Zerotier provide an easy eay to link devices together in a Virtual network. They are an easy way to allow access to devices without requiring much networking expertise. However, they require devices that support their apps in order to function.
+
+### Tailscale
+
+[Tailscale](https://tailscale.com/) is a VPN solution based on [WireGuard](https://www.wireguard.com/). The free tier of its service allows up to 100 connected devices. Tailscale is easy to setup but support is limited.
```

Comment:
Why no section about wireguard? it seems to be the only accountless free software

Reply:
OpenVPN is also an option.  A lot of FOSS, and even non-FOSS, router firmware includes an OpenVPN server.

Reply:
I'm not familiar with setting up these things, so I'm not including it in this guide. Maybe someone else can add it in a future PR.

Reply:
I can definitely do router based OpenVPN documentation.  Would you prefer a separate PR or can I add to this branch?

Reply:
I think its totally fine for wireguard to just link thier docs
https://www.wireguard.com/quickstart/

Reply:
Linked Wireguard quick start guide.